com.github.emmanueltouzery.cigale.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The metainfo, desktop file and the icon should be upstreamed.

Reply:
Upstreamed, you mean put in the Cigale repo itself? In fact they're already there (although the metainfo there is an old version that enables local build). I'll remove them from this repo and use those from the Cigale repo.